src/main/java/com/faforever/client/units/RackoversDBSettings.java

```diff
@@ -0,0 +1,44 @@
+package com.faforever.client.units;
+
+import lombok.Data;
+
+import java.util.Locale;
+
+@Data
+public class RackoversDBSettings {
+  private final String[] showArmies = {"Aeon", "UEF", "Cybran", "Seraphim"};
+  private final String previewCorner = "None";
+  private final String spookyMode = "0";
+  private final String lang;
+
+  public RackoversDBSettings(Locale locale) {
+    switch (locale.getLanguage()) {
+      case "cs":
+        lang = "CZ";
+        break;
+      case "de":
+        lang = "DE";
+        break;
+      case "es":
+        lang = "ES";
+        break;
+      case "fr":
+        lang = "FR";
+        break;
+      case "it":
+        lang = "IT";
+        break;
+      case "pl":
+        lang = "PL";
+        break;
+      case "ru":
+        lang = "RU";
+        break;
+      case "tzm":
+        lang = "TZM";
+        break;
+      default:
+        lang = "US";
+    }
+  }
```

Comment:
What happens if we pass an unknown language to the unit DB? I'd expect it to just fall back to US. In which case we could simply write:

```
locale.getLanguage().toUppercase(Locale.US)
```

Otherwise, I propose a list of supported languages like so:
```
private static final Collection<String> SUPPORTED_LANGUAGES = Arrays.asList("CZ", "DE", ...);
...
String uppercaseLanguage = locale.getLanguage().toUppercase(Locale.US));
return SUPPORTED_LANGUAGES .contains(uppercaseLanguage) ? uppercaseLanguage : "US";
```

Reply:
The only problem with that approach is that the database (the forged alliance game files) do not use those ISO 639-1 country codes. Also there was another reason why I created a switch statement having todo sth with changing ISO codes, but can't remember that right now.

Reply:
My solution above does the exact same thing as the switch case but it's way shorter :)